src/snapshot/tosvg.js

```diff
@@ -16,6 +16,9 @@ var Drawing = require('../components/drawing');
 var Color = require('../components/color');
 
 var xmlnsNamespaces = require('../constants/xmlns_namespaces');
+var DOUBLEQUOTE_REGEX = /"/g;
+var DUMMY_SUB = 'TOBESTRIPPED';
```

Comment:
is there any way this can occur in other contexts in the SVG string? Or a way to add characters that make it more obviously impossible?